nomad/nextcloud.nomad

```diff
@@ -0,0 +1,261 @@
+job "nextcloud" {
+  datacenters = ["dc1"]
+  type = "service"
+  
+  group "nc" {
+    task "nextcloud" {
+      driver = "docker"
+      config {
+        image = "nextcloud:15"
+        volumes = [
+          "/opt/hoover/volumes/nextcloud:/var/www/html",
+          "/opt/hoover/collections/ncsync:/var/www/html/data/ncsync/files",
```

Comment:
In the other nomad jobs, volumes are stored under `/var/local/liquid/volumes/{job}/{group-or-task}`, we should keep a consistent place. Maybe we even make it configurable someday.

Reply:
Configurable would be best. I just hacked it together like this because I didn't want to move all my files ^^
I didn't run it on any nomad cluster since I couldn't get Nomad to run in my VMs yet. I took your nomad files for the testdata collection and for hoover and just made a resembling file.

Reply:
What went wrong? We could do a debugging session after the call.